scanpy/external/tl/_gsea.py

```diff
@@ -0,0 +1,226 @@
+from typing import Optional, Tuple
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import scanpy.external as sce
+
+def gsea_barplot(
+    df: pd.DataFrame, 
+    score: Optional['str'] = 'nes',
+    score_cutoff: Optional[float] = None,
+    top_term: Optional[float] = 10,
+    figsize: Optional[Tuple[float, float]] = (6.5,6),
+    title: Optional['str'] = None,
+    x_label: Optional['str'] = None,
+    y_label: Optional['str'] = None,
+    out_dir: Optional['str'] = 'gseabarplot.png',
+    color = '#1f77b4', # matplotlib default: https://matplotlib.org/stable/users/dflt_style_changes.html#id1
+):
+  
+    """\
+    Horizontal barplot of GSEA results of genes by enrichment terms given a dataframe
+
+    Allows visualizations of either pval, nes, and fdr from outputs using either gseapy or fgsea.
+    Dataframe likely generated from scanpy.external.tl.gsea() function.
+    
+    Parameters
+    ----------
+    df
+        Dataframe where each row is an enrichment term with GSEA scores as columns
+    score
+        Chosen score to display. Default is 'nes'
+        Must be exact name of column as it appears in dataframe.
+        gseapy supports: 'es', 'nes', 'pval', 'fdr'
+        fgsea supports: 'es', 'nes', 'pval', 'padj', 'log2error'
+    score_cutoff
+        Cutoff for displayed values.
+        Example usage: for pval < 0.05
```

Comment:
Use case makes sense, but not exactly "example usage", maybe clearer wording?

Reply:
agreed, will fix!